logging/google/cloud/logging/logger.py

```diff
@@ -379,13 +410,17 @@ def log_text(self, text, labels=None, insert_id=None, severity=None,
 
         :type timestamp: :class:`datetime.datetime`
         :param timestamp: (optional) timestamp of event being logged.
+
+        :type resource :class:``google.cloud.logging.resource`
+        :param resource: (Optional) Monitored resource of the entry
         """
         self.entries.append(
             ('text', text, labels, insert_id, severity, http_request,
-             timestamp))
+             timestamp, resource))
```

Comment:
It could be either...few reasons I think `_GLOBAL_RESOURCE` is best. The best reason to keep it `_GLOBAL_RESOURCE` is because it's backwards compatible, previously every entry was set to global. Setting to `_GLOBAL_RESOURCE` or `None` seems reasonable, `_GLOBAL_RESOURCE` seems a bit easier since you can't accidentally leave off the resource altogehter (by forgetting in in Batch). My thinking looking at the API was that both Entry and Batch have a Resource and Batch overrides Resource is a bit weird (as evidenced by all discussion here), I wanted to expose ability to set it in either places, but just setting all the Entry resources to global seems like a pretty reasonable default (as evidenced by it having been the default until now).